exotations/task_maps/task_map/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ add_definitions(-DEXOTICA_DEBUG_MODE)
 
 find_package(catkin REQUIRED COMPONENTS
   exotica
+  exotica_py
```

Comment:
My comment was targeting the other way around - now ``task_map`` depends on ``exotica_python`` being built. Makes sense, but we should make the building of the Python bindings conditional, i.e. only call ``pybind_add_module`` if we know Python is present and has fully been built?